lib/WWW/WebKit2.pm

```diff
@@ -327,6 +359,12 @@ sub is_loading {
     return 0;
 }
 
+sub handle_form_submission {
+    my ($self, $view, $request) = @_;
```

Comment:
Is view needed here?

Reply:
I will leave $view here to show which variable is given to handle_form_submission from webkit.

Reply:
Fine with me